nodejs-webpack-sources/nodejs-webpack-sources.spec

```diff
@@ -0,0 +1,48 @@
+%global npm_name webpack-sources
+%global enable_tests 1 
+
+%{?nodejs_find_provides_and_requires}
+
+Name: nodejs-%{npm_name}
+Version: 0.1.3
+Release: 1%{?dist}
+Summary: Source code handling classes for webpack
+License: MIT
+URL: https://github.com/webpack/webpack-sources#readme
+Source0: http://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+BuildRequires: nodejs-packaging
+BuildArch:  noarch
+
+%if 0%{?fedora} >= 19
+ExclusiveArch: %{nodejs_arches} noarch
+%else
+ExclusiveArch: %{ix86} x86_64 %{arm} noarch
+%endif
+
+%{?nodejs_find_provides_and_requires}
+
+%description
+%{summary}
+
+%prep
+%setup -q -n package
+
+%build
+%nodejs_symlink_deps --build
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr .editorconfig .eslintrc .gitattributes .jsbeautifyrc .npmignore .travis.yml README.md lib npm-debug.log.18166252 package.json test %{buildroot}%{nodejs_sitelib}/%{npm_name}
```

Comment:
There seems to be a bunch of cruft in here that we don't really need to package, remove at your own judgement